framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/EntityFrameworkCore/AbpDbContext.cs

```diff
@@ -314,7 +315,25 @@ protected virtual void HandleExtraPropertiesOnSave(EntityEntry entry)
                     continue;
                 }
 
-                entry.Property(property.Name).CurrentValue = entity.GetProperty(property.Name);
+                var entityProperty = entity.GetProperty(property.Name);
+                if (entityProperty == null)
+                {
+                    continue;
```

Comment:
But we also want to set null if `entity.GetProperty(property.Name)` returns null. Because we already checked `entity.HasProperty(property.Name)` before. So, if `entity.GetProperty(property.Name)` returns `null` that means it is intentionally set to `null`. It doesn't mean the property is missing. So, skipping if `entityProperty == null` makes a breaking change and breaks the current behavour. This would be better: would be better

````csharp
if (entityProperty == null)
                {
entry.Property(property.Name).CurrentValue = null;
                    continue;
}
````

Reply:
I updated the code and unit test.